src/transit_equity/geospatial/hex_to_shapely.py

```diff
@@ -0,0 +1,21 @@
+import geopandas as gpd
+from shapely import wkb
+import binascii
```

Comment:
We need to add these packages as the dependencies to the toml file, and as package specification to the yaml file.